imports/lib/utils.js

```diff
@@ -134,6 +134,41 @@ export function getPrevPageFromHistory () {
   return navigationHistory[navigationHistory.length - 1]
 }
 
+const isFullscreen = () => (
```

Comment:
These were moved to `utils.js` so that they could be shared by `fullScreenButton` and `player`